bokeh/server/tests/test_utils.py

```diff
@@ -26,25 +24,25 @@ def helper():
     return wait_until(helper)
 
 
-def wait_redis_gone(port):
-    def helper():
-        client = redis.Redis(port=port)
-        try:
-            client.ping()
-            return False
-        except redis.ConnectionError:
-            return True
-    return wait_until(helper)
+# def wait_redis_gone(port):
+#     def helper():
+#         client = redis.Redis(port=port)
+#         try:
+#             client.ping()
+#             return False
+#         except redis.ConnectionError:
+#             return True
+#     return wait_until(helper)
```

Comment:
This was due to an 'undefined name', because `redis` and `zmq` are not being imported (the imports were commented). To make pyflakes happy, we should either uncomment the import, or comment the functions.